eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -0,0 +1,81 @@
+parameters:
+  ServiceDirectory: 'sdk/...' #defaults to building all track 2 go packages
+
+stages:
+  - stage: Build
+    jobs:
+    - job: Test
+      displayName: Build and Test on
+      variables:
+        - template: ../variables/globals.yml
+      strategy:
+        matrix:
+          Linux_Go113:
```

Comment:
Go 1.14 has released, can we add it to the matrix?

Reply:
Let me try!

Reply:
Done. Turns out that the `GoTool` step is super intelligent and actually will download from the official sources!